file_organizer_v2/docs/phase-3/migration-guide.md

```
@@ -68,13 +68,13 @@ file-organizer config migrate --from-version 1.x
 
 # Manual migration
 # Copy old config and update
-cp ~/.config/file-organizer/config.yaml ~/.config/file-organizer/config-old.yaml
+cp config/file-organizer/config.yaml config/file-organizer/config-old.yaml
 file-organizer config create --defaults phase3
 ```
 
 **New Phase 3 Settings**:
 ```yaml
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)